aws-cs-eks/Aws.EksCluster.csproj

```diff
@@ -7,8 +7,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Pulumi.Aws" Version="1.30.0-preview" />
-    <PackageReference Include="Pulumi.Kubernetes" Version="1.6.0-preview" />
+    <PackageReference Include="Pulumi.Aws" Version="2.0.0" />
+    <PackageReference Include="Pulumi.Kubernetes" Version="2.0.0" />
```

Comment:
We should probably use wildcard versions, `2.*` etc which would resolve to the highest `2.x.x` stable release